bin/internal/run-zowe.sh

```diff
@@ -95,7 +95,7 @@ fi
 #Explorers may be present, but have a prereq on gateway, not desktop
 if [[ $LAUNCH_COMPONENT_GROUPS == *"DESKTOP"* ]]
 then
-  LAUNCH_COMPONENTS=zss,app-server,${LAUNCH_COMPONENTS} #Make app-server the first component, so any extender plugins can use its config
```

Comment:
This sacrifices z/os for linux. A conditional is needed somewhere isntead. A reasonable solution to me would be to have the conditional within zss' start.sh, such that it starts if on z/os, and issues an info or warning message when not, to tell people that it's being skipped and that if they need it, hopefully they have one set up on zos already.